wgpu-hal/Cargo.toml

```diff
@@ -154,20 +157,12 @@ bit-set = { workspace = true, optional = true }
 range-alloc = { workspace = true, optional = true }
 gpu-allocator = { workspace = true, optional = true }
 hassle-rs = { workspace = true, optional = true }
+# For core macros. This crate is also reexported as windows::core.
+windows-core = { workspace = true, optional = true }
```

Comment:
I feel like I'd prefer to just use `windows::core`.

Reply:
Perhaps the comment should be clarified instead: this is an upstream bug where `::windows_core::` is always emitted instead of `::windows::core::` where possible.

Thanks to `^::` we can't `pub(crate) use windows::core as windows_core;` where the interface macro is called.

Reply:
Ahhh, the macros themselves need this dependency. Gottcha. Probably worth clarifying the comment